ui/app/components/task-group-row.js

```diff
@@ -10,12 +11,22 @@ import { lazyClick } from '../helpers/lazy-click';
 @tagName('tr')
 @classNames('task-group-row', 'is-interactive')
 export default class TaskGroupRow extends Component {
+  @service can;
+  @service token;
```

Comment:
Tiny: it seems like this injection isn’t actually used?

Reply:
Eep, correct, I'll remove it.